docs/advanced/md.md

```diff
@@ -1,4 +1,53 @@
 # Molecular Dynamics
 
+Molecular dynamics (MD) is a computer simulation method for analyzing the physical movements of atoms and molecules. The atoms and molecules are allowed to interact for a fixed period of time, giving a view of the dynamic "evolution" of the system. In the most common version, the trajectories of atoms and molecules are determined by numerically solving Newton's equations of motion for a system of interacting particles, where forces between the particles and their potential energies are calculated using first-principles calculations (first-principles molecular dynamics, FPMD), or interatomic potentials and molecular mechanics force fields (classical molecular dynamics, CMD).
 
-### blablabla
\ No newline at end of file
+By setting `calculation` to be `md` or `sto-md`, ABACUS currently provides six different MD evolution methods, which is specified by keyword `md_type` in the `INPUT` file:
+
+  - -1: FIRE method
+  - 0: NVE ensemble
+  - 1: NVT ensemble with Nose Hoover Chain
+  - 2: NVT ensemble with Langevin thermostat
+  - 3: NVT ensemble with Anderson thermostat
+  - 4: MSST method
+
+Furthermore, ABACUS also provides a [list of keywords](./input_files/input-main.md#molecular-dynamics) to control relevant parmeters used in MD simulations.
+
+[Examples](https://github.com/deepmodeling/abacus-develop/tree/develop/examples/vdw/si2) of MD simulations are also provided.
+There are six INPUT files corresponding to six different MD evolution methods in the directory.
+For examlpe, `INPUT_0` shows how to employ the NVE simulation.
+
+To run any of the fix cases, users may enter the directory, copy the corresponding input file to `INPUT`, and run ABACUS.
+
+## FIRE
+
```

Comment:
[FIRE](https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.97.170201) (fast inertial relaxation engine) is a MD-based minimization algorithm. It introduces inertia into the equation of motion, and follows a MD trajectory that descents to an energy-minimum.